frontend/providers/imagehub/src/layout/index.tsx

```diff
@@ -0,0 +1,39 @@
+import useSessionStore from '@/stores/session';
+import { Nunito } from '@next/font/google';
+import clsx from 'clsx';
+import { useEffect, useState } from 'react';
+import { createSealosApp, sealosApp } from 'sealos-desktop-sdk';
+import styles from './index.module.scss';
+const nunito = Nunito({ subsets: ['latin'] });
+
+export default function Layout({ children }: any) {
+  const { setSession } = useSessionStore();
+  const [isLodaing, setIsLoading] = useState(true);
+  useEffect(() => {
+    return createSealosApp({
+      appKey: 'sealos-image-hub'
+    });
+  }, []);
+
+  useEffect(() => {
+    const initApp = async () => {
+      try {
+        const result = await sealosApp.getUserInfo();
+        setSession(result);
```

Comment:
When deploying apps along side with custom unique url domain, we do not want to store sessions under local storage for safety purpose.
Maybe getSession will just save user infos as document variables is enough.